### PR TITLE
Added List and navigation

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added List and navigation In MADEUI

### CODE REVIEW

File: components/Navigation/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    border-radius:5px;
+    font-weight: bold;
+    font-size: 1.1rem;
+    padding: 0 1rem;
+    
+  }
+  .navigation_items{
+    float:right;
+    display:flex;
+}
```
Comment (MiheerTamkhane) [inline components/Navigation/Navigation.css:19]: I thinks float is not good practice with flex. Other than that you can use align and justify properties to position them.

**Review by MiheerTamkhane — APPROVED** [review]
Looks good, one suggestion in list and navbar the font-weight is slightly much. try reducing little bit.

File: components/Navigation/Navigation.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply components/Navigation/Navigation.css:19]: Okay,will try to imrove it.Thankyou

Comment (srishti-maurya) [reply components/Navigation/Navigation.css:19]: you may also try `auto-margin`

**Review by srishti-maurya — APPROVED** [review]

File: components/Navigation/Navigation.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply components/Navigation/Navigation.css:19]: > you may also try `auto-margin`

okay..Thank for the suggestion

**Pull request merged**